api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -303,5 +310,8 @@ public void abortGenomicExtractionJob(DbWorkspace dbWorkspace, String jobId) thr
             cohortExtractionConfig.operationalTerraWorkspaceNamespace,
             cohortExtractionConfig.operationalTerraWorkspaceName,
             dbSubmission.get().getSubmissionId());
+
+    dbSubmission.get().setTerraStatusEnum(TerraJobStatus.ABORTING);
```

Comment:
nit: we technically don't have to set this here since the get call will always check before displaying to the user. Sure, it's possible that we'll have the submission recorded as "RUNNING" for a long time if the user doesn't check again but "ABORTING" also isn't correct since the status should move to "ABORTED" soon after.

I have a slight preference for dropping the setter here just so this function does one less thing and we don't make an assumption on what Terra will do after we call abort submission.

Reply:
It'll check, but it'll take time for the status to update in Terra. I did this so that the user gets feedback right away that clicking the 'abort' button has had an effect.